src/api/mock.ts

```diff
@@ -15,19 +15,23 @@ const Math100Items: CourseItem[] = [
 	{
 		name: 'Assignment 3',
 		weight: 5,
+		grade: undefined,
```

Comment:
If TS is required you to specify this key, we should change the CourseItem definition back to `grade?: Grade`